package.json

```
@@ -1,211 +1,211 @@
+		"build-client": "gulp",
+		"build-extensions": "webpack --config ./webpack.config.extensions.js",
+		"build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
+		"build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client && NODE_ENV=production yarn build-extensions",
+		"build-languages": "gulp languages",
+		"docker:compose": "yarn docker:env && docker-compose -f docker/docker-compose.yml",
+		"docker:up": "yarn docker:compose up",
+		"docker:ngrok": "yarn docker:compose -f docker/docker-compose-ngrok.yml",
+		"docker:ngrok-up": "yarn docker:ngrok up",
+		"docker:ngrok-down": "yarn docker:ngrok down",
+		"docker:stop": "yarn docker:compose stop",
+		"docker:down": "yarn docker:compose down",
+		"docker:build": "docker run -it --rm  -v ${PWD}:/usr/src/app -w /usr/src/app node yarn build",
+		"docker:build-image": "yarn docker:compose build",
+		"docker:wp": "yarn docker:compose exec wordpress wp --allow-root --path=/var/www/html/",
+		"docker:install": "yarn docker:compose exec wordpress bash -c \"/var/scripts/install.sh\"",
+		"docker:uninstall": "yarn docker:compose exec wordpress bash -c \"/var/scripts/uninstall.sh\"",
+		"docker:multisite-convert": "yarn docker:compose exec wordpress bash -c \"/var/scripts/multisite-convert.sh\"",
+		"docker:sh": "yarn docker:compose exec wordpress bash",
+		"docker:phpunit": "yarn docker:compose exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
+		"docker:tail": "yarn docker:compose exec wordpress bash -c \"/var/scripts/tail.sh\"",
+		"docker:clean": "yarn docker:compose down --rmi all -v && rm -rf docker/wordpress/* docker/wordpress/.htaccess docker/wordpress-develop/* docker/logs/* docker/data/mysql/*",
+		"docker:env": "node -e \"require('fs').createWriteStream( 'docker/.env', { flags: 'a' } );\"",
+		"lint": "gulp jshint && eslint --ext .js --ext .jsx ./*.js _inc/client extensions -c .eslintrc.js",
+		"php:compatibility": "composer php:compatibility",
+		"php:lint": "composer php:lint",
+		"php:autofix": "composer php:autofix",
+		"test-adminpage": "yarn test-client && yarn test-gui",
+		"test-client": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js",
+		"add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",
+		"test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js gui",
+		"test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js modules",
+		"test-dangerci-and-adminpage": "yarn danger ci && yarn test-adminpage",
+		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"{_inc,bin,extensions,modules,tools,tests}/**/*.{js,jsx,json}\" ./*.{js,json}"
```

Let's do it all, that's how pre-commit works.
```suggestion
		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\""
```